pepper-apis/src/main/java/org/broadinstitute/ddp/util/ConfigUtil.java

```
@@ -45,6 +45,20 @@ public static Integer getIntIfPresent(Config cfg, String key) {
         return cfg.hasPath(key) ? cfg.getInt(key) : null;
     }
 
+    /**
+     * Get value as integer if the key is present and value is not null.
+     * If key not present then return a default value.
+     */
+    public static int getIntIfPresent(Config cfg, String key, int defaultValue) {
+        try {
+            return cfg.hasPath(key) ? cfg.getInt(key) : defaultValue;
+        } catch (ConfigException e) {
+            String value = e instanceof ConfigException.WrongType ? "=" + cfg.getValue(key) : "";
```

I'm starting to think that catching an exception and returning a default like this is hiding a potential error. My opinion is we should fail-fast here so folks have to fix up the config.

---

Removed try/catch.